k8_cortx_cloud/destroy-cortx-cloud.sh

```diff
@@ -451,7 +455,7 @@ function helmChartCleanup()
     printf "########################################################\n"
     print_header=true
     local charts
-    charts="$(helm list | { grep '^consul\|^cortx\|^kafka\|^openldap\|^zookeeper' || true; })"
+    charts="$(helm list | { grep '^consul\|^cortx\|^kafka\|^openldap|^zookeeper' || true; })"
```

Comment:
Missing a slash between `openldap` and `zookeeper`.

Reply:
The rest of the PR looks good to me. I will merge and patch this in a subsequent PR due to the source here being in Walter's forked repo.

Reply:
```suggestion
    charts="$(helm list | { grep '^consul\|^cortx\|^kafka\|^openldap\|^zookeeper' || true; })"
```